templates/pulpcore-worker@.service.erb

```diff
@@ -14,7 +14,6 @@ WorkingDirectory=/var/run/pulpcore-worker-%i/
 RuntimeDirectory=pulpcore-worker-%i
 ExecStart=/usr/bin/rq worker \
           -w pulpcore.tasking.worker.PulpWorker \
-          -n reserved-resource-worker-%i@%%h \
```

Comment:
Did you mean to remove the name specification entirely here, rather than just removing the hostname from the worker name? Shouldn't we keep names like `reserved-resource-worker-1` and `reserved-resource-worker-2` ?

Reply:
@wbclark the name was removed as part of the changes. I'm mirroring this template: https://github.com/pulp/ansible-pulp/blob/master/roles/pulp-workers/templates/pulpcore-worker@.service.j2

Reply:
Great, thanks for the explanation, and the contribution. Will ack and merge.